Tests/Microsoft.AppCenter.Test.Windows/TaskExtension.cs

```diff
@@ -8,26 +8,11 @@ public static class TaskExtension
     {
```

Comment:
Hi, thanks for the changes. However since we use GetAwaiter().GetResult() directly in code, we should remove this utility.